roles/ceph-defaults/defaults/main.yml

```diff
@@ -214,6 +214,7 @@ ceph_iscsi_config_dev: true # special repo for deploying iSCSI gateways
 # a URL to the .repo file to be installed on the targets.  For deb,
 # ceph_custom_repo should be the URL to the repo base.
 #
+ceph_custom_key: https://server.domain.com/ceph-custom-repo-key.asc
```

Comment:
shouldn't it be commented out by default?

Reply:
good point!